src/pages/Chat.js

```diff
@@ -0,0 +1,22 @@
+const Chat = () => {
```

Comment:
ich persönlich bin ja tatsächlich für:
```suggestion
export default function Chat() {
```
Da müssten wir auf Jans Meinung warten.

Reply:
haha ja ich kann damit leben 👍